src/SFA.DAS.FindApprenticeshipTraining.UnitTests/Application/TrainingCourses/Queries/WhenGettingTrainingCourseList.cs

```diff
@@ -39,6 +43,240 @@ public class WhenGettingTrainingCourseList
+                                MaxEmployerLevyCap = notFundingPrice
+                            },
+                            new ApprenticeshipFunding
+                            {
+                                EffectiveFrom = DateTime.UtcNow.AddDays(-1),
+                                EffectiveTo = null,
+                                MaxEmployerLevyCap = fundingPrice
+                            }
+                        },
+                        StandardDates = new List<StandardDate>
+                        {
+                            new StandardDate
+                            {
+                                EffectiveFrom = DateTime.UtcNow.AddMonths(-1),
+                                LastDateStarts = null
+                            }
+                        }
+                        
+                    },
+                    new GetStandardsListItem
+                    {
+                        Title = "Available",
+                        ApprenticeshipFunding = new List<ApprenticeshipFunding>
+                        {
+                            new ApprenticeshipFunding
+                            {
+                                EffectiveFrom = DateTime.UtcNow.AddDays(-10),
+                                EffectiveTo = DateTime.UtcNow.AddDays(9),
+                                MaxEmployerLevyCap = fundingPrice
+                            },
+                            new ApprenticeshipFunding
+                            {
+                                EffectiveFrom = DateTime.UtcNow.AddDays(4),
+                                EffectiveTo = null,
+                                MaxEmployerLevyCap = notFundingPrice
+                            }
+                        },
+                        StandardDates = new List<StandardDate>
+                        {
+                            new StandardDate
+                            {
+                                EffectiveFrom = DateTime.UtcNow.AddMonths(-1),
+                                LastDateStarts = null
+                            }
+                        }
+                        
+                    }
+                }
+            };
+            mockApiClient
+                .Setup(client => client.Get<GetStandardsListResponse>(
+                    It.Is<GetStandardsListRequest>(c=>c.Keyword.Equals(query.Keyword) && c.RouteIds.Equals(query.RouteIds))))
+                .ReturnsAsync(standardsListResponse);
+            mockApiClient
+                .Setup(client => client.Get<GetSectorsListResponse>(It.IsAny<GetSectorsListRequest>()))
+                .ReturnsAsync(sectorsApiResponse);
+            
+            //Act
+            var result = await handler.Handle(query, CancellationToken.None);
+            
+            //Assert
+            Assert.IsTrue(result.Courses.ToList().TrueForAll(c=>c.MaxFunding.Equals(fundingPrice)));
+        }
```

Comment:
these tests are more to do with the GetStandardsListItem - put them on that and the tests will be much simpler.

Reply:
Good idea - I've moved those two tests over that are specific to the `GetStandardsListItem`